rtos/rtx/TARGET_CORTEX_M/RTX_CM_lib.h

```diff
@@ -580,6 +656,29 @@ __attribute__((naked)) void software_init_hook (void) {
   );
 }
 
+// Opaque declaration of _reent structure
+struct _reent;
+
+void __malloc_lock( struct _reent *_r )
```

Comment:
please fix the formatting : ``( struct _reent *_r )``, no spacing around ( ) - ``(struct _reent *_r)``